source/ed/types.h

```diff
@@ -224,6 +224,19 @@ struct Line : public Header, Nameable {
 
 };
 
+struct LineGroup : public Header, Nameable {
+    const static nt::Type_e type = nt::Type_e::LineGroup;
+    std::string name;
```

Comment:
you already have a name with the Nameable struct.

I'm not sure you want to inherit from this one, it's up to you.